app/models/physical_server/operations.rb

```diff
@@ -6,6 +6,25 @@ module PhysicalServer::Operations
   include_concern 'ConfigPattern'
   include_concern 'Lifecycle'
 
+  def remote_console_acquire_resource_queue(userid)
+    task_opts = {
+      :action => "Acquiring remote console file or url from a physical server with uuid #{ems_ref} for user #{userid}",
```

Comment:
Not for this PR, but should this be i18n'd?

Reply:
Probably, none of these have this so at least this is consistent :laughing: